dspace-api/src/test/java/org/dspace/content/VersioningWithRelationshipsTest.java

```diff
@@ -1,4203 +0,0 @@
-/**
- * The contents of this file are subject to the license and copyright
```

Comment:
why did you remove it?

Reply:
Because it is duplicate of VersioningWithRelationshipsIT.java